module/zfs/vdev.c

```diff
@@ -1167,12 +1170,17 @@ vdev_open_children(vdev_t *vd)
 	}
 	tq = taskq_create("vdev_open", children, minclsyspri,
 	    children, children, TASKQ_PREPOPULATE);
-
-	for (c = 0; c < children; c++)
-		VERIFY(taskq_dispatch(tq, vdev_open_child, vd->vdev_child[c],
-		    TQ_SLEEP) != 0);
-
-	taskq_destroy(tq);
+    
+    if(tq != NULL) { // managed to spawn thread(s)?
```

Comment:
Let's drop the comments here since it's clear that they're handling the tq == NULL case.  They could be useful if they instead explained _why_ this could happen.  Also please use `/* */` style comments instead of `//`.